x/auth/types/querier.go

```diff
@@ -0,0 +1,21 @@
+package types
```

Comment:
The pattern is to have a `querier/` sub-package in a module, not under `types/`

Reply:
resolved as per slack conversations

Reply:
fp4k [12:03 PM]
YUP
I will be, I resolved all the errors in gaia yesterday
p.s. what are your thoughts on use of a querier package as opposed to just having the queriers as a part of `types` + `keeper`? (edited) 

bez [12:05 PM]
hmmm should be a unique package or in keeper
what are your thoughts?
I generally like the idea of a separate package

bez [12:24 PM]
because the querier can be quite large
and/or complex

fp4k [12:25 PM]
yeah I mean, for the way things are currently structured, I think it makes sense to be in `keeper` + `types` (we need it to be in both for dependancy management) - however we could make a larger restructure which separates out more subpackages for specific module elements like querier. (edited) 

bez [12:26 PM]
ohhhh
so the param types & constructors in `types/` and the querier itself in `keeper/`? (edited) 

fp4k [12:26 PM]
exactly

bez [12:26 PM]
yeah
im OK with that
makes sense to me
ignore my comment then